docs/dorothy/dorothy-not-loading.md

```diff
@@ -76,3 +76,50 @@ Close the preferences and open a new terminal tab, now Dorothy should be loaded
 4. Save the settings.
 
 5. Use the menu bar `Termina: New Terminal` or the Command Palette `Terminal: Create New Terminal` to open a new terminal. It should now start as a login shell, loading Dorothy as expected. Validate this by running a Dorothy command, like `dorothy theme`, to ensure everything is functioning as intended.
+
+## Debian Gnome
+
+1. Select Edit ▸ Preferences.
+
+2. In the sidebar, click on the + button next to the Profiles label.
+
+3. Enter a name for the new profile. You can change this name later.
+
+4. Click Create to create the new profile.
+
+5. Navigate to command and select 'Run a custom command instead of my shell.'
+
+6. Type in 'bash -l' to use bash as your command (or another shell of your choosing with the login shell mode enabled).
+
+7. Click on the arrow next to the profile name (in the left hand sidebar).
+
+9. Select 'Set as default.'
+
+10. Save the profile and restart your terminal.
+
+Source: https://help.gnome.org/users/gnome-terminal/stable/pref-profiles.html.en
+
+## Debian Konsole (KDE)
+
+Note from KDE.org:
+
+Konsole doesn’t provide convenience for running login shell, because developers don’t like
+the idea of running login shell in a terminal emulator.
+
+Of course, users still can run login shell in Konsole if they really need to. Edit the profile in use and modify its command to the form of starting a login shell explicitly, such as "bash -l" and "zsh -l" Source: https://docs.kde.org/stable5/en/konsole/konsole/konsole.pdf
+
+If the benefits of configuring Konsole in this manner outweighs the cons for your use case, proceed with the steps below.
```

Comment:
This is a copy paste job, which ultimately is one opinion versus another, which just causes confusion. Just keep the instructions that are below and trim this section. In the talks about Dorothy, I believe I go into detail about how wrong, lazy, improper, and inefficient such an anti-login-shell opinion is on this matter... actually, Dorothy's opinion on this is already included at the top of this file:

> It could be that your terminal application is refusing to operate your shell as a login shell, which is how Dorothy (and many other applications) identify whether they should complete loading themselves. If Dorothy were to load itself on non-login shells then every shell command would also have to load Dorothy which would be be an unnecessary slow down to every shell command! By having Dorothy only load inside login shells, we allow everything to work perfectly.

Reply:
Removed in my update commit: https://github.com/bevry/dorothy/pull/262/commits/6d6bfba7159d102a3b1879b4bdacc92c2349346c